makefile

```diff
@@ -375,23 +386,48 @@ docs/matlabhelp.doc: docs/genmatlabhelp.sh matlab/*.sh matlab/*.docsrc matlab/*.
 # =============================== CLEAN UP ==================================
 
 clean: objclean pyclean
+ifneq ($(MINGW),ON)
 	rm -f $(STATICLIB) $(DYNLIB)
 	rm -f matlab/*.mex*
 	rm -f $(TESTS) test/results/*.out perftest/results/*.out
 	rm -f $(EXAMPLES) $(FE) $(ST) $(STF) $(GTT) $(GTTF)
 	rm -f perftest/manysmallprobs
 	rm -f examples/core test/core perftest/core $(FE_DIR)/core
+else
```

Comment:
all these escaped chars and slashes is a nightmare, but if that is how gnu make on WSL wants it, fine.

Reply:
Actually, we do not even need these changes for WSL compatibility, but only to 'run make' clean natively on Windows. I have tried my best not to require the WSL whenever possible.